linux_os/guide/system/software/sudo/sudo_require_reauthentication/oval/shared.xml

```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("'Ensure sudo timestamp_timeout is appropriate - sudo timestamp_timeout") }}}
+    <criteria comment="The timestamp_timeout should be configured" >
+      <criterion comment="check configuration in /etc/sudoers" test_ref="test_sudo_timestamp_timeout" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist" comment="check correct configuration in /etc/sudoers" id="test_sudo_timestamp_timeout" version="1">
+    <ind:object object_ref="obj_sudo_timestamp_timeout"/>
+    <ind:state state_ref="state_sudo_timestamp_timeout" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="obj_sudo_timestamp_timeout" version="1">
+    <ind:filepath>/etc/sudoers</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*Defaults[\s]+timestamp_timeout=([\d]+)$</ind:pattern>
+    <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
+  </ind:textfilecontent54_object>
+
+  <ind:textfilecontent54_state id="state_sudo_timestamp_timeout"
+  version="1">
+    <ind:subexpression datatype="string" operation="equals" var_ref="var_sudo_timestamp_timeout" />
+  </ind:textfilecontent54_state>
+
+  <external_variable id="var_sudo_timestamp_timeout" datatype="string" version="1" comment="Variable value for sudo timestamp_timeout" />
```

Comment:
Sorry for the confusion here, but the OVAL actually doesn't need to check for the variable. For example, if someone selects the variable to be `3_minutes`, these OVAL check will invalid `0, 1 and 2`, which should still pass. The variable in this case should be only used for the remediation as it's already being used by the template. You can revert the last commit.

Reply:
Reverted commit (a9e7f71).